app/lib/features/space/widgets/space_card.dart

```diff
@@ -1,202 +1,202 @@
-    super.key, this.displayImage,  this.title,  this.subtitle,
+  const SpaceCard({
+    super.key,
+    this.displayImage,
+    this.title,
+    this.subtitle,
   });
 
   @override
   Widget build(BuildContext context) {
-    return LayoutBuilder(
-      builder: (context, constraints) {
-        if(constraints.maxWidth < 600){
-          return Container(
-          decoration: BoxDecoration(
-            color: Theme.of(context).colorScheme.primary,
-            borderRadius: BorderRadius.circular(10),
-          ),
-          width: 300,
-          child: Padding(
-            padding: const EdgeInsets.all(16.0),
-            child: Column(
-              children: [
-                 SquareAvatar(  
-                  height: 76,
-                  width: 77,
-                  displayImage: displayImage,
-                ),
-                Text(
-                  title!,
-                  style: Theme.of(context).textTheme.titleMedium,
-                ),
-                 Text(
-                  subtitle!,
-                  style: const TextStyle(color: Color(0xff79747E)),
-                ),
-                const SizedBox(
-                  height: 10,
-                ),
-                Row(
-                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                  children: const [
-                    SpaceCardIcons(
-                      icon: Atlas.ticket_plus,
-                      title: 'Invite',
-                    ),
-                    SpaceCardIcons(
-                      icon: Atlas.link_chain,
-                      title: 'Copy link',
-                    ),
-                    SpaceCardIcons(
-                      icon: Atlas.bell_dots,
-                      title: 'Notifications',
-                    )
-                  ],
-                ),
-                const Divider(
-                  thickness: 0.5,
-                  indent: 1,
-                  endIndent: 1,
-                ),
-                Container(
-                  margin: const EdgeInsets.only(top: 10, bottom: 10),
-                  child: Row(
-                    children: [
-                      Container(
-                          margin: const EdgeInsets.only(right: 10),
-                          child: const Icon(Atlas.padlock),),
-                      const Text('Encryption Settings')
-                    ],
+    final bool isDesktop =
+        desktopPlatforms.contains(Theme.of(context).platform);
+
+    return !isDesktop
+        ? Container(
+            decoration: BoxDecoration(
+              color: Theme.of(context).colorScheme.primary,
+              borderRadius: BorderRadius.circular(10),
+            ),
+            width: 300,
+            child: Padding(
+              padding: const EdgeInsets.all(16.0),
+              child: Column(
+                children: [
+                  ActerAvatar(
+                    mode: DisplayMode.Space,
+                    uniqueId: '',
```

Comment:
needs the same as above, but roomId and displayName don't seem to present here yet...

```suggestion
                    uniqueId: roomId,
                    displayName: displayName,
                    avatar: displayImage
```